blocks/library/freeform/freeform-block.js

```diff
@@ -287,7 +365,19 @@ export default class FreeformBlock extends wp.element.Component {
 			focus && (
 				<BlockControls
 					key="controls"
-					controls={ this.mapControls( FREEFORM_CONTROLS ) }
+					controls={ concat( this.mapControls( FREEFORM_CONTROLS ), {
+						leftDivider: true,
+						icon: 'ellipsis',
+						title: __( 'More' ),
+						isActive: showMore,
+						onClick: this.toggleMoreDrawer,
+						children: (
+							showMore && <div className={ moreDrawerClasses }>
+								<div className="more-draw__arrow" />
+								<Toolbar controls={ this.mapControls( MORE_CONTROLS ) } />
+							</div>
+						),
+					} ) }
```

Comment:
Nevermind, I see how it can be done now - I had assumed that the controls prop were the only way to set the content of the <BlockControls> element.